cmd/version.go

```diff
@@ -24,3 +24,9 @@ func MockVersion(version string) (restore func()) {
 	Version = version
 	return func() { Version = old }
 }
+
+// ProgramName represents the name of the application binary (i.e. pebble)
```

Comment:
Let's drop the i.e. and parenthesis. That's already clear from the default in the code. Same below.

```suggestion
// ProgramName represents the name of the application binary.
```